doc/user/content/demos/business-intelligence.md

```diff
@@ -325,12 +323,16 @@ Materialize to maintain for us.
 
 1. Modify the size of the **query01** card, and then click **Save**.
 
-1. Click **Auto-refresh**, and then select **1 minute** (which is the fastest
-   refresh rate that the tool offers; Materialize can reasonably support 1
-   second refreshes for many query types).
+1. Click **Auto-refresh**, and then select **1 minute.** (60 seconds is the
+   fastest refresh rate you can select in the UI, but if you copy the URL,
+   open a new tab and edit the end of the url to change the `refresh=60`
+   parameter to `refresh=1` you can force metabase to update every second.)
```

Comment:
Nit, but I think the tip would read slightly better if it were its own paragraph and not in a parenthetical!